py_experimenter/experimenter.py

```diff
@@ -99,7 +99,7 @@ def __init__(self,
 
         self.database_credential_file_path = database_credential_file_path
         if not self._is_valid_configuration(self.config, database_credential_file_path):
-            raise InvalidConfigError('Invalid configuration')
+            raise InvalidConfigError('Invalid configuration. See log messages for details')
```

Comment:
'Invalid configuration, see previous log messages for details.'

Reply:
I changed this